### DISCUSSION TITLE
Venice map in RE 2.0

### DISCUSSION THREAD

**Discussion by Crooked-Mind** [redeclipse/base#1192] [Maps and Content Showcase]
i ported this map, and it was here (basic old version)
[venice RE.zip](https://github.com/redeclipse/world/files/4220597/venice.RE.zip)

**Comment by cloudziq** [comment]
hmm, the map seems incomplete.  Like just a retextured version, there are many issues:
 -map is not saved after /calclight, it's not optimized :/ (610k verts VS 117k after /calclight)
 -lights are not set at all, seems like just placed randomly and there are too many of them :O
 -environment vars are just all defaults too, lighting is very basic, seems like only /sunlight was used from all possible world vars
 -there hundreds of active lights and other entities under the map
 -lack of alternate variant


You may have warned that this is only the base map or WIP version ;p   Currently your post seems like its a fully ported, finished version

**Comment by Crooked-Mind** [comment]
[venice RE.zip](https://github.com/redeclipse/world/files/4224926/venice.RE.zip)
![veniceRE](https://user-images.githubusercontent.com/61201938/74832718-b699de80-5339-11ea-90d4-8ae994366ae0.png)
new version of map: 
- some work with lights
- entity deleted under the map
- some works with environment  
- map saved after /calcight 
but i think, it looks like a WIP version too
